packages/authorization-token/src/db/authorization-token.ts

```diff
@@ -179,14 +181,16 @@ export const remove = async (
     );
   }
 
-  const dbToken = await prisma.authorizationToken.delete({
-    where: {
-      token_projectId: {
-        projectId: props.projectId,
-        token: props.token,
-      },
-    },
-  });
+  const dbToken = await context.postgrest.client
+    .from("AuthorizationToken")
+    .delete()
+    .eq("projectId", props.projectId)
+    .eq("token", props.token)
+    .select()
```

Comment:
Not an issue, just found that we don't use result of remove anywhere so can skip select too

Reply:
Anyway it will throw if result is not "single" so it's ok